api/client/client.go

```diff
@@ -275,7 +275,9 @@ func (c *Client) PushBucketPath(
 				Payload: &pb.PushBucketPathRequest_Chunk{
 					Chunk: buf[:n],
 				},
-			}); err != nil {
+			}); err == io.EOF {
+				break
```

Comment:
:sweat_smile:

Reply:
Oh yeah haha. I can remove that when digging into the nginx-related. steam error I mentioned in Slack.